content/articles/2021-08-01-dbee-nvim.md

```diff
@@ -0,0 +1,72 @@
+# My first vim plugin!
+
+Have you ever had the necessity of check a query while you are writing a python
+wrapper? If so, this post is for you.
+
+Many times I had the necessity of checking a query while I'm writing a function,
+debugging or testing it. That's why I wrote my first vim (well, actually a
+nvim) plugin called [`dbee.nvim`](https://github.com/mmngreco/dbee.nvim).
+
+## My using case
+
+I have a virtual environment with `SQLAlchemy` and others Python dependencies
+and then I want to check a query from my project. So, I have to open `DBeaver`
+or another SQL client copy-paste the query there and see the result. This
+process is very annoying.
```

Comment:
I have a virtual environment with `SQLAlchemy` and others Python dependencies,
and then I want to check a query from my project. So, I have to open `DBeaver`,
or another SQL client, copy-paste the query there and see the result. This 
process can be very annoying.